.travis.yml

```diff
@@ -46,6 +46,7 @@ matrix:
 before_script:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
+  - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
   - composer config -q github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
no you don't. ``composer config -g`` configures system-wide (well, for this user, not for the whole system). It is not the same than ``composer global config``, which would just change the config of the "global" project (``composer global`` is just changing the working directory to a different one and run composer commands on this special project)